### PR TITLE
[configure.ac] Remove --enable-loadedllvm

### PR DESCRIPTION
**Pull request by luhenry**
Fixes https://github.com/mono/mono/issues/8290

### CODE REVIEW

**Comment by vargaz** [comment]
As I mentioned, this will make development more painful. Also, on ios, we have 2 cross compilers, both with a libmonollvm.dylib, which are going to have a name collision, or they have to be lipo-d together.

**Comment by jaykrell** [comment]
I have only used the non-loaded form, but only a little.

File: configure.ac [hunk 1]
```diff
@@ -3219,9 +3219,8 @@ dnl ***  LLVM  ***
 dnl **************
 
 AC_ARG_ENABLE(llvm,[  --enable-llvm	Enable the LLVM back-end], enable_llvm=$enableval, enable_llvm=default)
-AC_ARG_ENABLE(loadedllvm,[  --enable-loadedllvm	Load the LLVM back-end dynamically], enable_llvm=$enableval && enable_loadedllvm=$enableval, enable_loadedllvm=no)
```
Comment (akoeplinger) [inline configure.ac]: autoconf will just issue a warning if someone passes this now unknown option right?

Comment (akoeplinger) [reply configure.ac]: We still have --enable-loadedllvm in packaging/MacSDK/mono.py and README.md

**Comment by luhenry** [comment]
Too many drawbacks for little removed code. We will revisit whenever we have a good story of use of in-tree builds of our libraries.

**Pull request closed without merging**